xrdsst/controllers/service.py

```diff
@@ -163,6 +164,57 @@ def list_services(self):
 
         self.list_service_description_services(active_config, self.app.pargs.client, self.app.pargs.description)
 
+    @ex(help="Delete service descriptions", arguments=[(['--ss'], {'help': 'Security server name', 'dest': 'ss'}),
+                                                       (['--client'], {'help': 'Client id', 'dest': 'client'}),
+                                                       (['--description'], {'help': 'Service description id', 'dest': 'description'})])
+    def delete_descriptions(self):
+        active_config = self.load_config()
+
+        if self.app.pargs.ss is None:
+            self.log_info('Security-server parameter is required for deleting service descriptions')
+            return
+
+        if self.app.pargs.client is None:
+            self.log_info('Client parameter is required for deleting service descriptions')
+            return
+
+        if self.app.pargs.description is None:
+            self.log_info('Description parameter is required for deleting service descriptions')
+            return
+
+        self.delete_service_descriptions(active_config, self.app.pargs.ss, self.app.pargs.client, self.app.pargs.description)
+
+    @ex(help="Update service descriptions", arguments=[(['--ss'], {'help': 'Security server name', 'dest': 'ss'}),
+                                                       (['--client'], {'help': 'Client id', 'dest': 'client'}),
+                                                       (['--description'], {'help': 'Service description id', 'dest': 'description'}),
+                                                       (['--code'], {'help': 'REST service code', 'dest': 'code'}),
+                                                       (['--url'], {'help': 'Service description url', 'dest': 'url'})])
+    def update_descriptions(self):
+        active_config = self.load_config()
+
+        if self.app.pargs.ss is None:
```

Comment:
Maybe here and in the other commands is better to remove the return on each "if" and just put an error flag or something, in that case, we could show the full list of missing parameters instead of one by one